### ISSUE TITLE
Komada cannot fetch messages after reboot

### ISSUE THREAD

**Issue by YorkAARGH** [UnseenFaith/komada#28]
![discordcanary_2016-10-19_12-20-38](https://cloud.githubusercontent.com/assets/20838878/19516981/88768480-95f6-11e6-8701-2c92012ec173.png)
As you can see by the screenshot above, when I try to pin a message that was posted BEFORE the bot was rebooted AFTER said bot rebooted, it cannot fetch the message.

**Comment by CyberiumShadow** [comment]
What's the underlying code?

Is it using `<TextChannel>.fetchMessage(id)`?

**Comment by YorkAARGH** [comment]
I actually don't know, the code I'm using in it's fullness

``` javascript
exports.run = (client, msg, [message]) => {
  msg.channel.sendMessage(`:pushpin: **${message.author.username}#${message.author.discriminator}** in #${message.channel.name} - ${require("moment")(message.timestamp).format("D[/]M[/]Y [@] HH:mm:ss")}\n${message.cleanContent}`);
};

exports.conf = {
  enabled: true,
  guildOnly: false,
  aliases: ["note"],
  permLevel: 0,
  botPerms: [],
  requiredFuncs: []
};

exports.help = {
  name: "pin",
  description: "Pin messages to a set channel.",
  usage: "<messageid:msg>",
  usageDelim: ""
};
```

**Comment by CyberiumShadow** [comment]
What about the function that runs that command?

**Comment by YorkAARGH** [comment]
I _think_ it's the following from `./functions/inhibitors/usage.js`, it's the only instance I could find that looks remotely like it could be doing what it is.

``` javascript
          case "msg":
          case "message":
            if (msg.channel.messages.has(args[i])) {
              args[i] = msg.channel.messages.get(args[i]);
            } else if (currentUsage.type === "optional" && !repeat) {
              args.splice(i, 0, undefined);
            } else {
              reject(`${currentUsage.possibles[0].name} must be a valid message id.`);
            }
            break;
```

But please don't quote me on that.

**Comment by bdistin** [comment]
Should be resolved in d3d498c99d5eca98b5cbcefb9838fa7d96f17c93

Let me know if you have any more troubles...
